hw3/03-react/src/navigation/Navbar.js

```diff
@@ -0,0 +1,32 @@
+import { Route, Routes, Link } from "react-router-dom";
+import Home from "./../views/Home";
+import Search from "./../views/Search";
+import Houses from "./../views/Houses";
+
+function Navbar() {
+  return (
+    <div>
+      <nav className="navbar">
+        <ul className="navbar">
+          <li className="nav-link mx-2 p-3">{<Link to="/">Home</Link>}</li>
+          <li className="nav-link mx-2 p-3">
+            {<Link to="/search">Search</Link>}
+          </li>
+          <li className="nav-link mx-2 p-3">
+            {<Link to="/houses">Houses</Link>}
+          </li>
+        </ul>
+      </nav>
+      <Routes>
+        <Route path="/" element={<Home title="Welcome to GoT!" />} />
+        <Route path="/search" element={<Search title="Search a Character" />} />
+        <Route
+          path="/houses"
+          element={<Houses title="Game of Thrones Houses" />}
+        />
+      </Routes>
+    </div>
+  );
+}
+
+export default Navbar;
```

Comment:
Why are the Home, Search and Houses components children of the Navbar component? The Navbar component should be used to build the navbar, not the majority of the site.